bangazonapi/views/order.py

```diff
@@ -140,7 +147,7 @@ def list(self, request):
             ]
         """
         customer = Customer.objects.get(user=request.auth.user)
-        orders = Order.objects.filter(customer=customer)
+        orders = Order.objects.filter(customer=customer).exclude(payment_type_id=None)
```

Comment:
More efficient query would be generated with 

```py
orders = Order.objects.filter(customer=customer, payment_type__isnull=False)
```

Reply:
this has been updated!